fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java

```diff
@@ -1642,13 +1640,19 @@ private void getNewImage(HostInfo helperNode) throws IOException {
         try {
             String hostPort = NetUtils.getHostPortInAccessibleFormat(helperNode.getHost(), Config.http_port);
             String infoUrl = "http://" + hostPort + "/info";
-            StorageInfo info = getStorageInfo(infoUrl);
+            ResponseBody<StorageInfo> responseBody = MetaHelper
+                    .doGet(infoUrl, HTTP_TIMEOUT_SECOND * 1000, StorageInfo.class);
```

Comment:
Is this should be `StorageInfoV2`?

Reply:
ResponseBody<StorageInfo> Equivalent to StorageInfoV2,so we do not need StorageInfoV2